__tests__/chat.test.js

```diff
@@ -185,15 +187,23 @@ describe('Chat Methods', () => {
     })
   })
 
-  describe('Chat createChannel, joinChannel and leaveChannel', () => {
+  describe('Chat createChannel, joinChannel, watchChannel, and leaveChannel', () => {
     it('Successfully performs the complete flow', async () => {
       const teamChannel = {
         name: config.teams.acme.teamname,
         public: false,
-        topic_type: 'chat',
-        members_type: 'team',
-        topic_name: 'subchannel',
+        topicType: 'chat',
+        membersType: 'team',
+        topicName: 'subchannel',
+      }
+      const generalChannel = {
+        name: config.teams.acme.teamname,
+        public: false,
+        topicType: 'chat',
+        membersType: 'team',
+        topicName: 'general',
```

Comment:
It'd be nice to have a few different types of channel here, like different team, topic type, membersType, etc.